server/api/batches.py

```diff
@@ -82,9 +82,27 @@ def serialize_batch(batch: Batch) -> JSONDict:
             }
             for tally_sheet in batch.result_tally_sheets
         ],
+        "lastEditedBy": construct_batch_last_edited_by_string(batch),
     }
 
 
+def construct_batch_last_edited_by_string(batch: Batch) -> Optional[str]:
+    if batch.last_edited_by_support_user_email:
+        return batch.last_edited_by_support_user_email
+    if batch.last_edited_by_user_type == UserType.JURISDICTION_ADMIN:
+        return batch.last_edited_by_user_key  # Email
+    if batch.last_edited_by_user_type == UserType.TALLY_ENTRY:
+        tally_entry_user = TallyEntryUser.query.filter_by(
+            id=batch.last_edited_by_user_key
+        ).one_or_none()
+        if tally_entry_user is not None:
```

Comment:
One edge case that I've been thinking through is how to handle the case that the relevant tally entry user no longer exists, say because they were deleted.

This concern doesn't exist for deleted jurisdiction admins because we'll always have an email in `last_edited_by_user_key`.